pkg/reconciler/kubequota/kubequota_controller.go

```diff
@@ -328,7 +312,7 @@ func (c *Controller) startQuotaForClusterWorkspace(ctx context.Context, clusterN
 				logger.V(4).Info("got API change notification")
 
 				ctx, discoveryCancel = context.WithCancel(ctx)
-				resourceQuotaController.UpdateMonitors(ctx, c.dynamicDiscoverySharedInformerFactory.DiscoveryData)
+				resourceQuotaController.UpdateMonitors(ctx, c.dynamicDiscoverySharedInformerFactory.ServerPreferredNamespacedResources)
```

Comment:
Needs to be ServerPreferredResources because we support quota of cluster-scoped too

Reply:
Done with fd037fb2723512a41e381cc80829722cdc9d59d8.